--- common/include/pcl/common/impl/eigen.hpp.old	Fri Aug 28 18:39:51 2015
+++ common/include/pcl/common/impl/eigen.hpp	Fri Aug 28 19:02:09 2015
@@ -866,7 +866,8 @@
                            pcl::ModelCoefficients::Ptr plane_out,
                      const Eigen::Transform<Scalar, 3, Eigen::Affine> &transformation)
 {
-  Eigen::Matrix < Scalar, 4, 1 > v_plane_in (std::vector < Scalar > (plane_in->values.begin (), plane_in->values.end ()).data ());
+  std::vector < Scalar > vals (plane_in->values.begin (), plane_in->values.end ());
+  Eigen::Matrix < Scalar, 4, 1 > v_plane_in (&vals[0]);
   pcl::transformPlane (v_plane_in, v_plane_in, transformation);
   plane_out->values.resize (4);
   for (int i = 0; i < 4; i++)
