--- test/geometry/CMakeLists.txt.old	Sat Aug 29 20:39:41 2015
+++ test/geometry/CMakeLists.txt	Sat Aug 29 20:40:08 2015
@@ -6,9 +6,9 @@
              FILES test_mesh_circulators.cpp
              LINK_WITH pcl_gtest)
 
-PCL_ADD_TEST(geometry_mesh_conversion test_mesh_conversion
-             FILES test_mesh_conversion.cpp
-             LINK_WITH pcl_gtest pcl_common)
+#PCL_ADD_TEST(geometry_mesh_conversion test_mesh_conversion
+#             FILES test_mesh_conversion.cpp
+#             LINK_WITH pcl_gtest pcl_common)
 
 PCL_ADD_TEST(geometry_mesh_data test_mesh_data
              FILES test_mesh_data.cpp
