--- outofcore/include/pcl/outofcore/impl/octree_base_node.hpp.old	Sat Aug 29 16:19:50 2015
+++ outofcore/include/pcl/outofcore/impl/octree_base_node.hpp	Sat Aug 29 20:25:05 2015
@@ -426,7 +426,7 @@
         {
           root_node_->m_tree_->incrementPointsInLOD (this->depth_, p.size ());
           
-          payload_->insertRange (p.data (), p.size ());
+          payload_->insertRange (&p[0], p.size ());
           
           return (p.size ());
         }
@@ -444,7 +444,7 @@
           if (!buff.empty ())
           {
             root_node_->m_tree_->incrementPointsInLOD (this->depth_, buff.size ());
-            payload_->insertRange (buff.data (), buff.size ());
+            payload_->insertRange (&buff[0], buff.size ());
 //            payload_->insertRange ( buff );
             
           }
