--- outofcore/include/pcl/outofcore/visualization/outofcore_cloud.h.old	Sat Aug 29 16:27:05 2015
+++ outofcore/include/pcl/outofcore/visualization/outofcore_cloud.h	Sat Aug 29 16:27:09 2015
@@ -1,7 +1,16 @@
 #ifndef PCL_OUTOFCORE_OUTOFCORE_CLOUD_H_
 #define PCL_OUTOFCORE_OUTOFCORE_CLOUD_H_
 
-#include <stdint.h>
+#include <boost/cstdint.hpp>
+using boost::uint8_t;
+using boost::int8_t;
+using boost::int16_t;
+using boost::uint16_t;
+using boost::int32_t;
+using boost::uint32_t;
+using boost::int64_t;
+using boost::uint64_t;
+using boost::int_fast16_t;
 
 // PCL
 //#include <pcl/common/time.h>
