--- test/features/test_spin_estimation.cpp.old	Sat Aug 29 12:03:49 2015
+++ test/features/test_spin_estimation.cpp	Sat Aug 29 12:04:02 2015
@@ -254,7 +254,7 @@
       cloud_xyzi.points.push_back (p);
     }
   }
-  cloud_xyzi.width = static_cast<uint32_t> (cloud_xyzi.points.size ());
+  cloud_xyzi.width = static_cast<boost::uint32_t> (cloud_xyzi.points.size ());
 
   // Compute the intensity-domain spin features
   typedef Histogram<20> IntensitySpin;
