--- test/features/test_rift_estimation.cpp.old	Sat Aug 29 11:59:56 2015
+++ test/features/test_rift_estimation.cpp	Sat Aug 29 12:00:25 2015
@@ -65,12 +65,12 @@
       cloud_xyzi.points.push_back (p);
     }
   }
-  cloud_xyzi.width = static_cast<uint32_t> (cloud_xyzi.points.size ());
+  cloud_xyzi.width = static_cast<boost::uint32_t> (cloud_xyzi.points.size ());
 
   // Generate the intensity gradient data
   PointCloud<IntensityGradient> gradient;
   gradient.height = 1;
-  gradient.width = static_cast<uint32_t> (cloud_xyzi.points.size ());
+  gradient.width = static_cast<boost::uint32_t> (cloud_xyzi.points.size ());
   gradient.is_dense = true;
   gradient.points.resize (gradient.width);
   for (size_t i = 0; i < cloud_xyzi.points.size (); ++i)
