--- test/search/test_kdtree.cpp.old	Sat Aug 29 14:23:19 2015
+++ test/search/test_kdtree.cpp	Sat Aug 29 14:23:30 2015
@@ -56,7 +56,7 @@
     for (float y = -0.5f; y <= 0.5f; y += resolution)
       for (float x = -0.5f; x <= 0.5f; x += resolution)
         cloud.points.push_back (PointXYZ (x, y, z));
-  cloud.width = static_cast<uint32_t> (cloud.points.size ());
+  cloud.width = static_cast<boost::uint32_t> (cloud.points.size ());
   cloud.height = 1;
 
   cloud_big.width = 640;
