--- test/features/test_pfh_estimation.cpp.old	Sat Aug 29 11:35:23 2015
+++ test/features/test_pfh_estimation.cpp	Sat Aug 29 11:48:19 2015
@@ -558,7 +558,7 @@
         p.z = static_cast<float> (z);
         cloud->points.push_back (p);
       }
-  cloud->width = static_cast<uint32_t> (cloud->points.size ());
+  cloud->width = static_cast<boost::uint32_t> (cloud->points.size ());
   cloud->height = 1;
 
   GFPFHEstimation<PointXYZL, PointXYZL, GFPFHSignature16> gfpfh;
