--- src/cpp/flann/util/serialization.h
+++ src/cpp/flann/util/serialization.h
@@ -89,7 +89,9 @@
 BASIC_TYPE_SERIALIZER(float);
 BASIC_TYPE_SERIALIZER(double);
 BASIC_TYPE_SERIALIZER(bool);
-
+#ifdef _MSC_VER
+BASIC_TYPE_SERIALIZER(unsigned __int64);
+#endif


 // serializer for std::vector
@@ -117,6 +119,7 @@
     }
 };

+
 // serializer for std::vector
 template<typename K, typename V>
 struct Serializer<std::map<K,V> >
